### DISCUSSION TITLE
ghcr image and tags don't match docker hub.

### DISCUSSION THREAD

**Discussion by cwiggs** [seerr-team/seerr#1181] [Ideas]
I noticed that the ghcr image tags are mostly missing, there is 1 for `develop` and no other tags.  It would be nice to update the workflow so that ghcr and docker hub had the same images and image tags.  Can someone point me to the Github action that pushes to docker hub so we can add ghcr to that workflow as well?

Thanks

**Comment by gauthier-th** [comment]
Related to #736 I think
